### ISSUE TITLE
Can’t recover wallets

### ISSUE THREAD

**Issue by Cympullmyndyd** [berry-pool/nami#877]
I got hacked and can’t recover 2 accounts out of 13 I had. When I deleted extension and entered seed phrase to change password etc. 11 accounts showed up but 2 did not. Wanted to see if there was anything that could be done? This is the wallet. 1900 Ada got transferred out but the nfts are still in it.
addr1q9gxkshn3e7k20ugka03yxxayaflxjgchm026jfas9yh74yu5jjyesztg8ww7sl4j9wkudz5lxcs9pevqf2ajgjyvc3slk3sxx

**Comment by Saraluvv** [comment]
@Cympullmyndyd  Make use of the Nami web app [click here ](https://app.namiweb.xyz) to connect and import

**Comment by Riley-Kilgore** [comment]
@Cympullmyndyd no, unfortunately once something like this has happened, the damage is done.

Multisigs and other technology like hardware wallets can help prevent hacks from occuring, but once something has been done, it cannot be undone.

Do not fall for scam links.

**Comment by Bendy127** [comment]
So sorry about what happened same thing happened to me my wallet got hacked and about 0.9 btc was stolen thanks to Mr John a whitehathacker who helped me recover all of it message him on telegram too t.me/Whitehathacker217
